sysutils/bd/Portfile

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+# Using the latest commit, since there was
+# a fix for Mac OSX after the release.
+github.setup        vigneshwaranr bd c497fe7d
+
+version             1.0.2
+revision            0
+
+categories          sysutils
+platforms           darwin
+license             MIT
+maintainers         nomaintainer
+
+description         a tool to cd into parent directories, instead of ../../.., etc
+long_description    Quickly go back to a specific parent directory in bash \
+                    instead of typing \"cd ../../..\" redundantly.
+
+checksums           rmd160  7c65faa1995763309454b2ab5e879aa150af9697 \
+                    sha256  37e72b32d8dcb65951c7239a1c665fcd9efee144299405621b65c42ecfd2caaa \
+                    size    143089
+
+depends_lib-append  port:bash-completion
+
+use_configure       no
+build {}
+
+destroot {
+    set bash_completion ${destroot}/${prefix}/share/bash-completion/completions
+    set completion_dir  ${prefix}/share/${name}/completion
+    set doc_dir         ${destroot}${prefix}/share/doc/${name}
+
+    xinstall -d -m 755  ${bash_completion} ${destroot}${completion_dir} ${doc_dir}/screenshot
+    xinstall -m 755     ${worksrcpath}/${name} ${destroot}${prefix}/bin/${name}
+    xinstall -m 644     ${worksrcpath}/bash_completion.d/${name} ${destroot}${completion_dir}
+    xinstall -m 644     ${worksrcpath}/screenshot/${name}.png ${doc_dir}/screenshot
+    xinstall -m 644 -W  ${worksrcpath} LICENSE README.md ${doc_dir}
+
+    ln -s ${completion_dir}/${name} ${bash_completion}/${name}
+}
+
+notes "
+::
+:: If this is a new install... To complete the
```

Comment:
Please remove the extraneous `:: ` formatting at the beginning of each line; allow MacPorts to format the notes as it usually does.

Please add a backslash, without a preceding space, at the end of any lines (like the one ending `To complete the`) so that MacPorts can do word wrapping as appropriate to the user's terminal width.

Reply:
Ok, I'd guess that goes for cgit as well. I've seen other ports using hash marks instead (#). I find the double :: a cleaner approach. Thought it was ok.

Done √

Reply:
In general, ports should not add extraneous formatting to the notes. It gives the impression of attempting to draw additional attention to one port's notes over another port's notes, which is not generally justified. Instead, ports should write simple paragraphs of notes and allow MacPorts to format them as it always does.